src/autopas/containers/LinkedCells.h

```diff
@@ -146,17 +159,29 @@ class LinkedCells : public ParticleContainer<Particle, ParticleCell, SoAArraysTy
   }
 
   bool isContainerUpdateNeeded() override {
+    std::atomic<bool> outlierFound{};
+    outlierFound = false;
```

Comment:
move initialization to constructor of outlierFound

Reply:
fixed
dc1d8e0